### DISCUSSION TITLE
Site that require cookie for premium content

### DISCUSSION THREAD

**Discussion by tathastu871** [deniscerri/ytdlnis#87] [Q&A]
Had a premium account for a site and how can i downlaod videos from that site how to supply cookies to ytdlnis app.
on commandline we can supply cookie but how to same in this app

**Comment by deniscerri** [comment]
Get the beta on telegram. It has cookie support. 
App will be released soon with that feature

**Reply by prateekpatel1432** [reply]
No can't do this

**Reply by deniscerri** [reply]
> No can't do this

The latest version already has cookies. Go to the more section and press cookies.

Then press add cookie and write the website url. Log in and when press ok. That's it. The app will use that from now on 

U only have to do it once

**Comment by prateekpatel1432** [comment]
To download video or audio download video player to play from u tube
